calculator.rb

```diff
@@ -0,0 +1,98 @@
+#Setting Up Operators
+operators = ["1", "add", "+", "2", "subtract", "-", "3", "multiply", "*", "4", "divide", "/", "5", "exponent", "^", "6", "modulo", "remainder", "%"]
+numbers = []
+
+#Validation Methods
```

Comment:
It is a great practice to put this functionality into it's own method. Nice work.